Chatkit/Repositories/Filters/StateFilter.swift

```diff
@@ -0,0 +1,8 @@
+
+protocol StateFilter {
+    
+    func hasModifiedSubstate(oldState: VersionedState, newState: VersionedState) -> Bool
+    func hasCompleteSubstate(_ state: VersionedState) -> Bool
+    func hasSupportedSignature(_ signature: VersionSignature) -> Bool
```

Comment:
Not certain that "Supported" is the correct verb here.  Is "Relevant"  a more suitable alternative?

Reply:
I will rename this method.

Reply:
Thanks